README.md

```diff
@@ -109,6 +109,10 @@ will need to do this binding so the image gets built when maven is run from the
       </executions>
     </plugin>
 
+To remove the image run the following command:
+
+	mvn docker:removeImage
```

Comment:
@krystiannowak The user is required to supply the image name here right? eg `-DimageName=foobar`?